web/src/beta/components/ActionItem/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from "@emotion/styled";
+import { FC } from "react";
+
+import Icon from "../Icon";
+
+type Props = {
+  icon: string;
+  title: string;
+  onClick?: () => void;
+};
+
+const ActionItem: FC<Props> = ({ icon, title, onClick }) => {
+  return (
+    <Box onClick={onClick}>
+      <Icon icon={icon} style={{ width: "8px", height: "8px" }} />
```

Comment:
Icon has a `size` prop you can use. Please use that instead of setting the size through inline css 🙏🏻